_posts/2023-12-18-omero-5-6-10.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.10
+intro-blurb: The OME team is pleased to announce the release of OMERO.server 5.6.10
+---
+
+Today we are releasing OMERO.server 5.6.10 which includes:
+
+- an upgrade of logback-classic to version 1.3.14
+- an upgrade of Bio-Formats to version 7.1.0, see the [release announcement]({{ site.baseurl }}/2023/12/11/bio-formats-7-1-0.html).
+
+
+**Note:** This upgrade will invalidate the Bio-Formats Memoizer cache. Please
+see the upgrade guide for further information.
+
+
+### Installing the Software:
+
+For full details of the changes included in the OMERO 5.6 series see the
+[OMERO 5.6.0]({{ site.baseurl }}/2020/01/15/omero-5-6-0.html) release
```

Comment:
This is unlucky. This link when clicked leads you to 5.6.0 release announcement. There is clearly stated that "currently supported platforms" are CentOS7, Ubuntu 18.04 .....
![Screenshot 2023-12-18 at 11 09 34](https://github.com/ome/www.openmicroscopy.org/assets/2478303/37c0aa69-c47e-4fbc-bf0f-2267f2da8794)

Reply:
thanks for spotting that and adding the clarification